packages/grafana-data/src/types/datasource.ts

```diff
@@ -416,6 +416,8 @@ export type ExploreQueryFieldProps<
 
 export interface QueryEditorHelpProps<TQuery extends DataQuery = DataQuery> {
   datasource: DataSourceApi<TQuery>;
+  query: TQuery;
+  refId: string;
```

Comment:
you can also remove the refId from here (it can be accessed with `query.refId`)

Reply:
Perfect, thanks!